test/systemtest/test_separate_pkgs.py

```diff
@@ -161,6 +161,13 @@ def test_pkg_one_correct_one_license_file_missing(self):
         self.assertTrue(check_output_status(
             stdout, SUCCESS, WARNING, FAILURE, FAILURE, SUCCESS))
 
+    def test_pkg_scheme3_conform(self):
+        """Test on a package that has all attributes for being conform to
+        the official scheme v3"""
+        process, stdout = open_subprocess("test_pkg_scheme3_conform")
+        self.assertEqual(os.EX_OK, process.returncode)
+        self.assertTrue(check_output_status(stdout))
```

Comment:
I think we also need tests that explicitly conform with scheme 1 and one for 2. 
And also at least one negative test each, i.e. that don't comply with the schemas.

Reply:
This is implemented in feature/add_config_params, is this ok for you or do you want me to copy the tests in here too?